### PR TITLE
Add Contributor Covenant 1.4 copy

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Review by NeilHanlon — APPROVED** [review]
looks good to me

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # OpenELA Governance Documentation
 
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
```
Comment (mstg) [inline README.md]: Shouldn't the badge say 1.4 instead of 2.1?

**Review by mstg — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dirkmueller) [reply README.md]: changed

**Review by mstg — APPROVED** [review]

**Comment by mstg** [comment]
Thanks!

**Pull request merged**